mu/modes/circuitpython.py

```diff
@@ -24,46 +24,24 @@
 from mu.interface.panes import CHARTS
 
 
-class AdafruitMode(MicroPythonMode):
+class CircuitPythonMode(MicroPythonMode):
     """
-    Represents the functionality required by the Adafruit mode.
+    Represents the functionality required by the CircuitPython mode.
     """
 
-    name = _('Adafruit CircuitPython')
-    description = _("Use CircuitPython on Adafruit's line of boards.")
-    icon = 'adafruit'
-    save_timeout = 0  #: Don't autosave on Adafruit boards. Casues a restart.
-    connected = True  #: is the Adafruit board connected.
+    name = _('CircuitPython')
+    description = _("Boards running Circuit Python.")
+    icon = 'circuitpython'
+    save_timeout = 0  #: Noautosave on CP boards. Will restart.
+    connected = True  #: is the board connected.
     force_interrupt = False  #: NO keyboard interrupt on serial connection.
     valid_boards = [
-        (0x239A, 0x8015),  # Adafruit Feather M0 CircuitPython
-        (0x239A, 0x8023),  # Adafruit Feather M0 Express CircuitPython
-        (0x239A, 0x801B),  # Adafruit Feather M0 Express CircuitPython
-        (0x239A, 0x8014),  # Adafruit Metro M0 CircuitPython
-        (0x239A, 0x8019),  # Adafruit CircuitPlayground Express CircuitPython
-        (0x239A, 0x801D),  # Adafruit Gemma M0
-        (0x239A, 0x801F),  # Adafruit Trinket M0
-        (0x239A, 0x8012),  # Adafruit ItsyBitsy M0
-        (0x239A, 0x8021),  # Adafruit Metro M4
-        (0x239A, 0x8025),  # Adafruit Feather RadioFruit
-        (0x239A, 0x8026),  # Adafruit Feather M4
-        (0x239A, 0x8028),  # Adafruit pIRKey M0
-        (0x239A, 0x802A),  # Adafruit Feather 52840
-        (0x239A, 0x802C),  # Adafruit Itsy M4
-        (0x239A, 0x802E),  # Adafruit CRICKit M0
-        (0x239A, 0xD1ED),  # Adafruit HalloWing M0
-        (0x239A, 0x8030),  # Adafruit NeoTrellis M4
-        (0x239A, 0x8032),  # Grand Central
         (0x2B04, 0xC00C),  # Particle Argon
         (0x2B04, 0xC00D),  # Particle Boron
         (0x2B04, 0xC00E),  # Particle Xenon
-        (0x239A, 0x8034),  # future board
-        (0x239A, 0x8036),  # future board
-        (0x239A, 0x8038),  # future board
-        (0x239A, 0x803A),  # future board
-        (0x239A, 0x803C),  # future board
-        (0x239A, 0x803E),  # future board
         (0x239A, None),    # Any Adafruit Boards
+        (0x1209, 0x2017),  # Mini SAM M4
+        (0x1209, 0x7102),  # Mini SAM M0
```

Comment:
Since this PR is about renaming the mode and this is a new addition, would you mind opening a new PR for this one?
To be fair, this PR is not that large and since it's already here it doesn't matter that much, your call if you prefer to leave it.

Reply:
No problem - I can remove this and create a new PR.